lib/beaker-answers/answers.rb

```diff
@@ -55,32 +76,42 @@ def self.supported_versions
     # @param [Array<Beaker::Host>] hosts An array of host objects.
     # @param [Hash] options options for answer files
     # @option options [Symbol] :type Should be one of :upgrade or :install.
+    # @option type [Symbol] :format Should be one of :bash or :hiera
+    #   Setting :bash will result in the "classic" PE answer file being generated
+    #   Setting :hiera will generate the new PE hiera config file format
     # @return [Hash] A hash (keyed from hosts) containing hashes of answer file
     #   data.
-    def self.create version, hosts, options
+    def self.create(version, hosts, options, type = :bash)
```

Comment:
We do typically interface with the Answers lib through that options hash, so it does seem like a good idea to specify the type there. On versions > 20162 we could just ignore that value?